### DISCUSSION TITLE
I want to make a replica(read-only) of serverless v2 using cdk.

### DISCUSSION THREAD

**Discussion by deathquin** [aws/aws-cdk#24374] [General]
### Describe the issue

I am using golang-based aws cdk.
I am satisfied with serverless v2 in the database.
However, I want to make a read-only through cdk, but I can't find it in the document and I can't find it in the search. I want to test the efficiency by adding read-only.
So I want some help. Even if it is not Golang, I would appreciate it if you let me know based on typescript.
I want you to help me.

Thnak you

### Links

https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds.DatabaseInstanceReadReplica.html

**Comment by pahud** [comment]
Moving to discussion as this seems to be a general guidance request.

Can you elaborate more about what you are trying to do in your use case?
